### DISCUSSION TITLE
PdfMatrix raises DeprecationWarning now

### DISCUSSION THREAD

**Discussion by stumpylog** [ocrmypdf/OCRmyPDF#1192] [Q&A]
Just a heads up that as of pikepdf [8.7.0](https://github.com/pikepdf/pikepdf/blob/main/docs/releasenotes/version8.rst#v870) using PdfMatrix is now raising a DeprecationWarning in favor of pikepdf.Matrix

**Comment by jbarlow83** [comment]
Fixed in ocrmypdf 15.4.3.
